harness/tests/requirements/requirements-harness.txt

```diff
@@ -5,10 +5,10 @@ mypy==0.910
 responses!=0.23.2
 requests_mock
 coverage
-pytorch_lightning==1.5.9
 deepspeed==0.8.3
 transformers>=4.8.2,<4.29.0
 attrdict3
 moto
 # Pydantic V2 has changes that break existing tests
 pydantic<2
+setuptools==59.5.0
```

Comment:
hm?  What's this for?

Reply:
lint-python was failing after I removed `pytorchlightning==1.5.9` requirement. I checked `pip freeze` step for diffs and saw that `setuptools` was a newer version than what we have in `main` right now. I pinned `setuptools` to that version that we are currently using. 

https://app.circleci.com/pipelines/github/determined-ai/determined/43092/workflows/ca21f339-12d1-4816-a8a1-53836bef3d8e/jobs/1711190

Reply:
nvm, thought this was setup.py.  This is fine.

But, please leave a comment explaining why you need this pin, and when it will be safe to remove (that is, when we upgrade our CI from torch 1.9 to torch 1.11+ or whatever)